astropy/time/tests/test_delta.py

```diff
@@ -188,6 +188,8 @@ def test_mul_div(self):
         dt5 = self.dt * np.arange(3)
         assert dt5[0].jd == 0.
         assert dt5[-1].jd == (self.dt + self.dt).jd
+        dt6 = self.dt * [0, 1, 2]
+        assert np.all(dt6.jd == dt5.jd)
```

Comment:
I see that other cases are now working as expected (yay):
```
In [2]: dt = TimeDelta(1, format='sec')

In [7]: dt / [1]
Out[7]: <TimeDelta object: scale='None' format='sec' value=1.0>

In [8]: [1] * dt
Out[8]: <TimeDelta object: scale='None' format='sec' value=1.0>

In [9]: [1] / dt
Out[9]: <Quantity [86400.] 1 / d>
```
I don't see these in the test cases (though possible I missed seeing), so if not you should add them for completeness and preventing future regressions.

Reply:
good point, added.